tests/console/ncurses.pm

```diff
@@ -26,7 +26,7 @@ sub run {
     # Try to draw a bold red line
     script_run('echo "$(tput smacs;tput setaf 1;tput bold)lqqqqqqqqqqqqqqk$(tput rmacs;tput sgr0)"');
     # Try a simple yes/no dialog
-    script_run 'dialog --yesno "test for boo#1054448" 3 20 | tee boo_1054448.tee';
+    script_run 'dialog --yesno "test for boo#1054448" 3 20 | tee boo_1054448.tee', die_on_timeout => 0;
```

Comment:
Using `script_run()` here is wrong. The dialog needs to be executed using `enter_cmd()` instead.

Reply:
Fixed